includes/rest/class-following-controller.php

```diff
@@ -91,24 +104,30 @@ public function get_items( $request ) {
 		 */
 		\do_action( 'activitypub_rest_following_pre' );
 
-		$response = array(
-			'@context'  => get_context(),
-			'id'        => get_rest_url_by_path( \sprintf( 'actors/%d/following', $user->get__id() ) ),
-			'generator' => 'https://wordpress.org/?v=' . get_masked_wp_version(),
-			'actor'     => $user->get_id(),
-			'type'      => 'OrderedCollection',
-		);
+		$order    = $request->get_param( 'order' );
+		$per_page = $request->get_param( 'per_page' );
+		$page     = $request->get_param( 'page' ) ?? 1;
+		$context  = $request->get_param( 'context' );
 
-		/**
-		 * Filter the list of following urls.
-		 *
-		 * @param array                   $items The array of following urls.
-		 * @param \Activitypub\Model\User $user  The user object.
-		 */
-		$items = \apply_filters( 'activitypub_rest_following', array(), $user );
```

Comment:
With this removal, can `__construct()` and `default_following()` be removed as well?